Resources/Private/Translations/de/Modules.xlf

```diff
@@ -163,6 +163,14 @@
                 <source>The source path doesn't match the expression {0}</source>
         <target>Der Quellpfad entspricht nicht dem Muster {0}</target>
       </trans-unit>
+      <trans-unit id="error.targetUriPathNotValid" xml:space="preserve">
+        <source>The target path doesn't match the expression {0}</source>
+        <target>Der Zielpfad entspricht nicht dem Muster {0}</target>
+      </trans-unit>
+      <trans-unit id="error.hostNotValid" xml:space="preserve">
+        <source>The host doesn't match the expression {0}</source>
```

Comment:
The expression should be wrapped with quotes to make it easier to read with its special characters.
We should also adjust that for existing labels.